static/styles.css

```diff
@@ -0,0 +1,58 @@
+* {
```

Comment:
Please use sass which is automatically compiled with Zola. See https://www.getzola.org/documentation/content/sass/

Reply:
I'll rewrite it to `scss` as it is closer to `css` than `sass`. Is that okey?

Reply:
Ok